README.md

```diff
@@ -5,3 +5,7 @@ Barefoot Nomad - Making company travel and accomodation easy and convinient.
 Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.
 
 ---
+
+## Project Status
+
+[![Reviewed by Hound](http://img.shields.io/badge/Reviewed%20By-Hound-%23a874d1)](https://houndci.com)
```

Comment:
Nice work Samuel. But, most badges that I often see are always at the top of README.md file. Why did you put it here?

Reply:
Thank you for your review, The reason why I put the badge at the top of the readme and why I think others do similar is so that at first glance of the repo you may know the status of the project and attached services like Hound.